test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -27,10 +38,18 @@ def create_and_join_group_chat(device_1_home, device_2_home, chat_name):
 
     # device 2: open group chat
     device_2_chat = device_2_home.get_chat_with_user(chat_name).click()
-    device_2_chat.join_chat_button.click()
 
     return device_1_chat, device_2_chat
 
+def create_and_join_group_chat(device_1_home, device_2_home, chat_name):
+    device_1_chat, device_2_chat = create_new_group_chat(device_1_home, device_2_home, chat_name)
+    device_2_chat.join_chat_button.click()
+    return [device_1_chat, device_2_chat]
```

Comment:
instead of list, you can use tuple `device_1_chat, device_2_chat`

Reply:
as result we can use method in test like:
device_1_chat, device_2_chat = create_and_join_group_chat()